test/githubapp_incoming_test.go

```diff
@@ -11,19 +11,42 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/tektoncd/pipeline/pkg/names"
-	"gotest.tools/v3/assert"
-
 	"github.com/openshift-pipelines/pipelines-as-code/pkg/apis/pipelinesascode/v1alpha1"
 	tgithub "github.com/openshift-pipelines/pipelines-as-code/test/pkg/github"
 	"github.com/openshift-pipelines/pipelines-as-code/test/pkg/options"
 	"github.com/openshift-pipelines/pipelines-as-code/test/pkg/payload"
 	"github.com/openshift-pipelines/pipelines-as-code/test/pkg/secret"
 	"github.com/openshift-pipelines/pipelines-as-code/test/pkg/wait"
+	"github.com/tektoncd/pipeline/pkg/names"
+	"gotest.tools/v3/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func TestGithubIncomingWebhook(t *testing.T) {
+// all pipelineruns in .tekton directory have eventType as incoming
```

Comment:
usually you should have the name of the function first for go comment, it's strange that golangci-lint didn't bug you about it

Reply:
I thought the same way but i did not get the lint complain so i ignored :laughing:

Reply:
Updated comment in proper way

Reply:
what about trying to fix the linter next time